### DISCUSSION TITLE
Overlay driver audio transcript on that driver's onboard video

### DISCUSSION THREAD

**Discussion by toinestin** [f1multiviewer/issue-tracker#400] [Feature requests]
Now the audio transcript are getting very accurate it would be nice to add them to the driver onboards. Like is the case for the live timing. Perhaps like a subtitle view or even a dedicated message bar. Or in the short term if we could popout the audio transcripts window, and make the background transparent we can overlay it already manually.

**Comment by dbeckham** [comment]
At the very least, being able to pop out the messages would be fantastic

**Comment by bolderbast** [comment]
I would also SO MUCH like this to be an option. Each onboard window would show only the transcripts for that particular driver. I created a mockup of how it could look:

![MVF1 - AI Transrcipt mockup](https://github.com/f1multiviewer/issue-tracker/assets/78593849/de5d9cde-df3b-4b5c-acde-0d038bdba73b)
